src/pages/clients/show/components/Gateways.tsx

```diff
@@ -115,17 +110,30 @@ export function Gateways(props: Props) {
 
   useEffect(() => {
     if (deleteGatewayTokenId) {
-      setIsConfirmationVisible(true);
+      setIsModalVisible(true);
     }
   }, [deleteGatewayTokenId]);
 
   return (
     <>
-      <ConfirmActionModal
-        onClick={() => handleDeleteGatewayToken()}
-        onClose={() => setDeleteGatewayTokenId('')}
-        disabledButton={isFormBusy}
-      />
+      <Modal
```

Comment:
@beganovich Quick confirmation before your review. The reason I removed `ConfirmActionModal` is that when the component uses an atom, all other props are also overwritten if the same component is rendered elsewhere on the page. In this case, we use it for bulk deleting invoices, so it overwrites the disabled and other props. I discovered this behavior for the first time after encountering this bug :)